LL/corco_bascind.trig

```diff
@@ -85,26 +85,24 @@
     a foaf:Person;
     irishRel:genName "Decci";
     irishRel:nomName "Decc";
+    foaf:knows <http://www.wikidata.org/entity/Q165479> ;
```

Comment:
Yeah, many of the links in LL/corco_bascind.trig are of the form `foaf:knows <http://www.wikidata.org/entity/Q165479>;` so I left it that way.  I agree that we should link to the one and then from there link outwards so that if wikidata ever disappears, it will not break too many things.  If you do not mind, we can fix them all at once after this gets merged.

Reply:
Yup. That sounds like a good idea but something best done en masse.